pandas/tests/test_categorical.py

```diff
@@ -1673,6 +1673,10 @@ def test_unique(self):
         exp_cat = Categorical(["b", np.nan, "a"], categories=["b", "a"])
         tm.assert_categorical_equal(res, exp_cat)
 
+        # GH 18051 unique()._codes should be writeable
```

Comment:
you just need to compare the result of ``.unique()`` that's the user visible thing we are testing here. The non-writable issue is detail.